### PR TITLE
Update ir.m3u

### PR DESCRIPTION
**Pull request by xmha97**
Fix a channel link

### CODE REVIEW

File: channels/ir.m3u [hunk 1]
```diff
@@ -158,6 +158,6 @@ https://ir3.live21.ir/hls2/valiasr.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/7uEI6ck.png" group-title="News",VOA Persian
 https://voa-lh.akamaihd.net/i/voapnn_7@72817/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/v2wS68r.png" group-title="Religious",Nour TV
-https://184.75.213.27/liveevent.m3u8
+http://184.75.213.27/liveevent.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: I don't know exactly what the reason is, but this link doesn't work for me right now.
Captura de pantalla 2020-01-13 a las 16 04 10

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/ir.m3u (hunk 1 above)
Comment (amirrammari) [reply channels/ir.m3u]: Yes, https://streamtest.in/ can't test this URL but playing on players.
Are you can play this URL with VLC?

Comment (freearhey) [reply channels/ir.m3u]: No, I can't. It return HTTP error 400.
Captura de pantalla 2020-01-13 a las 17 02 21

Comment (amirrammari) [reply channels/ir.m3u]: OK, I think it's working only in my country.
Please remove it.

**Review by freearhey — APPROVED** [review]

**Pull request merged**